priv/templates/nimble_template/assets/.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "env": {
+    "browser": true,
+    "node": true,
+    "es2021": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:prettier/recommended"
```

Comment:
I'm not sure, with this change, do we still need the `prettier` addon? 🤔

Reply:
Yes @rosle although this config will install the `prettier` (if it isn't installed yet) but our prettier module contains more topics than just `install`, we have
- prettier-plugin-eex
- add prettier, prettier.fix commands, alias

So we still need the `prettier` addon module there FYU